rqsettings.py

```diff
@@ -2,14 +2,15 @@
 
 import sentry_sdk
 from sentry_sdk.integrations.rq import RqIntegration
+from sentry_sdk.integrations.redis import RedisIntegration
 
-if os.path.exists('config_local.py'):
+if os.path.exists("config_local.py"):
```

Comment:
Hmm we don't pylint this file right. I'd done that manually. This would fail pylint_quotes

Reply:
We don't pylint it, I'll fix it though